OpenRA.Game/Game.cs

```diff
@@ -36,6 +36,8 @@ public static class Game
 		public const int Timestep = 40;
 		public const int TimestepJankThreshold = 250; // Don't catch up for delays larger than 250ms
 
+		public static readonly Guid GameId = Guid.NewGuid();
```

Comment:
Again, the server code should avoid referring to anything in the Game class because dedicated servers run without a game client.  This doesn't change anything in practice, but is a code smell and potential maintenance issue.

Moving this to a `Platform.SessionGUID` field should be sufficient.